modules/data.atmosphere/R/download.CRUNCEP_Global.R

```diff
@@ -113,16 +116,16 @@ download.CRUNCEP <- function(outfolder, start_date, end_date, site_id, lat.in, l
       }
 
 
-      dat.list[[j]] <- ncdf4::ncvar_get(dap,
+      dat.list[[j]] <- retry.func(ncdf4::ncvar_get(dap,
                                  as.character(var$DAP.name[j]),
                                  c(lon_grid, lat_grid, 1),
-                                 c(1, 1, ntime))
+                                 c(1, 1, ntime)), maxErrors=maxErrors, sleep=sleep)
 
-      var.list[[j]] <- ncdf4::ncvar_def(name = as.character(var$CF.name[j]),
+      var.list[[j]] <- retry.func(ncdf4::ncvar_def(name = as.character(var$CF.name[j]),
```

Comment:
My understanding is that ncvar_def works locally (it sets up a variable in memory that can later be written to the output file), so maybe this one can stay unwrapped?